.github/workflows/ci.yml

```diff
@@ -11,17 +13,17 @@ jobs:
   build:
     runs-on: ${{ matrix.os }}
     strategy:
-      fail-fast: true
+      fail-fast: false
```

Comment:
Why?

Reply:
Because with `fail-fast: true`, if one fails, you do not get information about the status of the other matrix entries.
(It's like the old GHC, which could only give you _one_ error.)

Reply:
Sure, if GHA resources are a problem, but then I think the duplication of CI runs in PRs should also be addressed.

Reply:
Ok.